### DISCUSSION TITLE
Synchronization issue

### DISCUSSION THREAD

**Discussion by clickclack69** [GetPublii/Publii#1351] [General]
Hello,
I reopen this issue.
On my PC I have 2 users and each one uses Publii.
With one, no synchronization (S3) problem.
With the other, I launch the synchronization, the 1st phase happens _but it is very long_, Publii connects to my server (S3) and then the message **Uploading website** appears and nothing happens. I have to cancel the synchronization.

Deployment-process.log contains : 
[Sat, 10 Feb 2024 07:22:54 GMT] <- files.publii.json
[Sat, 10 Feb 2024 07:22:54 GMT] S3 ERROR: write ECONNABORTED
Is the message "write ECONNABORTED" the cause or the consequence of my Cancel ?

Rendering-deployment-process.log reflects the modifications I made.

Rendering-process.log contains no error.

All the xxx-errors.log are empty

When I restart the synchronization, the 1st phase is OK, Publii connects to my server (S3) and then the synchronization takes place, and all relevant files are transfered.

Could you please help me?

**Comment by dziudek** [comment]
Hi,

If it is exactly the same config the I suspect some networking issues on your side - it can be related to firewall rules, wifi connection etc.

Logs definitely suggests that when trying to download files.publi.json from bucket - there is some networking issue. In my opinion it is not a problem with Publii itself, but with your network or account configuration.
